models/classes/search/SearchProxy.php

```diff
@@ -106,10 +107,16 @@ public function search(ServerRequestInterface $request): array
         $queryParams = $request->getQueryParams();
         $results = $this->executeSearch($query);
 
-        $response = $this->getResultSetResponseNormalizer()
-            ->normalize($query, $results, $queryParams['params']['structure']);
+        if (
+            isset($queryParams['params']['rootNode'])
+            && in_array($queryParams['params']['rootNode'], self::SAFE_NODES)
```

Comment:
Please, use `strict` mode
```suggestion
            && in_array($queryParams['params']['rootNode'], self::SAFE_NODES, true)
```

Reply:
fixed, thanks! :partying_face: